libs/coin-modules/coin-polkadot/src/bridge/signOperation.ts

```diff
@@ -20,6 +20,7 @@ import type {
 import { encodeOperationId } from "@ledgerhq/coin-framework/operation";
 import { SignerContext } from "@ledgerhq/coin-framework/signer";
 import { buildTransaction, calculateAmount, getNonce, isFirstBond, signExtrinsic } from "../logic";
+import { shortenMetadata } from "../network/sidecar";
```

Comment:
Import from backend call should only be done from "../network", no direct call to subfile in network

Reply:
Do you mean that I should add "shortenMetadata" in "/ledger-live/libs/coin-modules/coin-polkadot/src/network/index.ts"?
I just have a look at other polkadot backend call.
If I don't make mistake, it seems that all the backend calls are in "network/sidecar" file. What do you think about that?